api/src/main/java/org/openmrs/module/registrationcore/api/impl/PatientEditListener.java

```diff
@@ -0,0 +1,20 @@
+package org.openmrs.module.registrationcore.api.impl;
+
+import org.openmrs.Patient;
+import org.openmrs.event.Event;
+import org.openmrs.module.registrationcore.RegistrationCoreConstants;
+
+import javax.jms.Message;
+
+public class PatientEditListener extends PatientActionListener {
```

Comment:
I'm surprised that this class is so much shorter than the creation one, e.g. doesn't use any proxy privileges. I guess that's because it's delegated to the provider.